src/Components/Voting.jsx

```diff
@@ -60,7 +60,7 @@ export const Voting = ({ setSingleReview, review_id, singleReview }) => {
         </button>
         Votes: {singleReview.votes}
```

Comment:
this votes state can currently be upvoted or downvoted many times, can this be ammended?